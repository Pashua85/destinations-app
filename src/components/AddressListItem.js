import React from 'react';

const AddressListItem = (props) => (
  <div>
    {props.item.address}
    <button className='remove-btn'>Удалить</button>
  </div>
)

export default AddressListItem;

