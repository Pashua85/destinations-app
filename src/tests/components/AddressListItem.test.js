import React from 'react';
import { shallow } from 'enzyme';
import AddressListItem from '../../components/AddressListItem';
import addresses from '../helpers/addresses';

test('should render AddressListItem with fixture data', () => {
  const wrapper = shallow(<AddressListItem item={addresses[0]} />);
  expect(wrapper).toMatchSnapshot();
});