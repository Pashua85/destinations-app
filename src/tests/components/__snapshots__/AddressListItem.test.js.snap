// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AddressListItem with fixture data 1`] = `
<div
  className="address-list__item"
>
  <div
    className="address-list__title"
  >
    ул. Щорса, 94А, Екатеринбург, Свердловская обл., Россия, 620144
  </div>
  <button
    className="address-list__btn remove-btn"
  >
    Delete
  </button>
</div>
`;
